paper/src/main/java/com/denizenscript/denizen/paper/properties/PaperWorldExtensions.java

```diff
@@ -23,6 +29,81 @@ public static void register() {
             return new ElementTag(world.getWorld().getNoTickViewDistance());
         });
 
+        if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_20)) {
+
+            // <--[tag]
+            // @attribute <WorldTag.gateway_count>
+            // @returns ElementTag(Number)
+            // @description
+            // Returns the number of end gateway portals.
+            // Only works in end worlds.

```

Comment:
Minor nitpick, but `in` sounds more like the player being in an end world while the script is running or something - maybe `on`, `for`, `if the world is an end world`, etc.?